Holovibes/sources/tools/tools.cc

```diff
@@ -37,7 +38,8 @@ std::string get_record_filename(std::string file_path, std::string prepend)
     std::string filename = filePath.filename().stem().string();
     std::string extension = filePath.extension().string();
     std::string path = filePath.parent_path().string();
-    std::filesystem::path oldFilePath = path + "/" + prepend + "_" + filename;
+    std::string eye = API.record.get_recorded_eye_file_string();
```

Comment:
Maybe add another parameter for setting a string After the filename ? It will avoid a call to the api from tool. And it will allow the api function to be deleted (since it's an internal function it shouldn't bé exposed to the user)